Player/Player.gd

```diff
@@ -0,0 +1,26 @@
+extends CharacterBody2D
+
+
+const speed = 200
+@onready var anim = get_node("AnimatedSprite2D")
```

Comment:
I like it in full becuse its easier for me to understand what the code is doing when reading it since im learning. Thank you for the tip but for now I prefer to leave it like this.